### PR TITLE
Update README with instructions

### PR DESCRIPTION
**Pull request by swalkinshaw**

### CODE REVIEW

**Comment by JulienMelissas** [comment]
Looks great! :shipit:

**Comment by austinpray** [comment]
Someone accept this PR

File: README.md [hunk 1]
```diff
@@ -1,5 +1,72 @@
...
+This project can be cloned and re-configured to fit your needs but we highly suggest you follow in the instructions below to create your own. This will guarantee you have the latest version of Bedrock, bedrock-ansible, Sage, and Soil in case this example falls behind a little.
+
+Here's how this example project was created:
+
+1. Create a new project directory: `$ mkdir example.com`
+2. Clone bedrock-ansible: `$ git clone --depth=1 git@github.com:roots/bedrock-ansible.git ansible`
+3. Move `Vagrantfile` to root: `$ mv ansible/Vagrantfile .`
+4. Clone bedrock: `$ git clone --depth=1 git@github.com:roots/bedrock.git site`
+5. Clone Sage: `$ cd site/web/app/themes && git clone --depth=1 git@github.com:roots/sage.git sage`
+6. Delete `.git` folders: `$ rm -rf ansible/.git && rm -rf site/.git && rm -rf site/web/app/themes/sage/.git`
```
Comment (QWp6t) [inline README.md]: There is disagreement between steps 5 and 6. In step 5, you CD into themes folder. In step 6, you're back in project root.

**Comment by QWp6t** [comment]
Also I'm thinking maybe `<environment>` and `<site name>` need to be explained a bit further, maybe with actual values used as examples.

I'm just going by that thread on discourse where the guy said we should explain _everything_.

**Pull request merged**